### PR TITLE
Add clickup raycast extension for browsing clickup tasks

### PR DESCRIPTION
**Pull request by Ali-Parandeh**
## Description

An extension for browsing click up tasks inside teams, spaces, folders and lists. Each tasks has a detail page where users can then read task description and click to go the click up task page from raycast.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/43654026/149643540-f805a5c6-ce2b-4ab1-91b8-d3ffbe5ea5ca.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

## Current Features

- [x] List and explore Your teams, spaces, folders, lists and tasks
- [x] Get a birds eye of number of tasks in each folder and list
- [x] List, search and preview contents of each task in Tasks View
- [x] View each tasks by priority
- [x] Open each task via the browser
- [x] Set your default team ID and space IDs

## Features to be Added (As standalone commands or as a single command group)
- [ ] Search and order all of your ClickUp Team and space tasks by any filters or query (e.g. due date, priority, status, duration in progress etc.)
- [ ] List and view tasks assigned to you ordered by priority and due date
- [ ] View and browse subtasks
- [ ] Copy task links to clipboard
- [ ] View assignees from Tasks view
- [ ] Set assignees, sprint scores, time estimates, priorities, tags and due dates from Raycast 
- [ ] Update status and priority of tasks from Raycast 
- [ ] Add subtasks and checklists from Raycast 
- [ ] Add Task dependencies from Raycast 
- [ ] Create folders, lists and tasks from Raycast
- [ ] List Tasks by sprints
- [ ] Search tasks comments from Raycast
- [ ] Search, view and download task attachments from Raycast
- [ ] Comment on tasks from Raycast
- [ ] Move tasks between lists and sprints from Raycast
- [ ] Delete and archive tasks from Raycast
- [ ] Assign Github PR IDs to your tasks from Raycast (Requires Github Token)
- [ ] Other cool features? Open an issue or PR!

### CODE REVIEW

File: extensions/clickup/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# ClickUp Tasks Explorer
+
+Plugin for Raycast to explore your [ClickUpTasks](https://github.com/Ali-Parandeh/raycast-clickup-extension).
+
+## How to obtain an API Token
+- Go to [ClickUp](https://clickup.com/login)
```
Comment (pernielsentikaer) [inline extensions/clickup/README.md:6]:
```suggestion
- Go to [ClickUp](https://app.clickup.com/login)
```

File: extensions/clickup/package.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "name": "manage-clickup-tasks",
+  "title": "Clickup Tasks Explorer",
+  "description": "Browse ClickUp tasks from RayCast",
+  "icon": "clickup-icon.png",
+  "author": "aparandeh",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Clickup Tasks Explorer",
```
Comment (pernielsentikaer) [inline extensions/clickup/package.json:12]:
```suggestion
      "title": "Tasks Explorer",
      "subtitle": "Clickup",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Ali-Parandeh 👋 

Thanks for an awesome extension! 💪 

I don't have a way to test it with data, do you by any chance have a token I could borrow for a few minutes?

One thing I noticed in the code is that when the API token is wrong it shows the toast, it should also stop the loader with an empty array – can you look into this?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by Ali-Parandeh** [comment]
@pernielsentikaer I created a playground for you

`pk_32613843_B7RVB0QF7USTO0FH76S0F31MZRGK9YLQ`

**Comment by Ali-Parandeh** [comment]
Now handling error states properly in views and hooks

**Review by pernielsentikaer — APPROVED** [review]
Hi @Ali-Parandeh 👋 

Awesome extension, and thanks a lot for the token 💯 Really well done 💪 I'm going to accept it and then will my colleague do a final review before it's finally merged.

**Comment by Ali-Parandeh** [comment]
Thank you @pernielsentikaer! That's exciting news :) I'll be working on it more so it'll be like Todoist extension in terms of functions.

**Pull request merged**